parsers/HOPS.py

```diff
@@ -7,13 +7,46 @@
 import logging
 import pandas as pd
 
-url = "https://www.hops.hr/Home/PowerExchange"
+from datetime import datetime
+
+URL = "https://www.hops.hr/Home/PowerExchange"
+
+
+def fetch_solar_production(feed_date, session=None, logger=logging.getLogger(__name__)):
+    """
+    Calls extra resource at https://files.hrote.hr/files/EKO_BG/FORECAST/SOLAR/FTP/TEST_DRIVE/<dd.m.yyyy>.json
+    to get Solar power production in MW.
+    :param feed_date: date_time string from the original HOPS feed
+    :return: Float
+    """
+    r = session or requests.session()
+
+    dt = datetime.strptime(feed_date, '%Y-%m-%d %H:%M:%S')
+    url_date = '{dt.day}.{dt.month}.{dt.year}'.format(dt=dt)
+    solar_url = 'https://files.hrote.hr/files/EKO_BG/FORECAST/SOLAR/FTP/TEST_DRIVE/{0}.json'.format(url_date)
+    response = r.get(solar_url)
+    obj = response.json()
+
+    df = pd.DataFrame.from_dict(obj['FullPower']).set_index('Timestamp')
+    solar_production_dt = datetime.strftime(dt, '%Y-%m-%d %H:00:00')
+    try:
+        solar = df['Value'].loc[solar_production_dt]
+        # Value is in string format and needs to be converted to float
+        solar = float(solar.replace(",", "."))
+        # Converting to MW
+        solar *= 0.001
+    except KeyError:
+        logger.info("No value for Solar power production on {0}".format(solar_production_dt))
+        solar = 0
```

Comment:
Probably needs to be `solar = None` here

Reply:
@systemcatch solar set to 0 since in Python3.6+ it's not allowed to make numeric operations with `None`. So line `unknown = df['value'].loc['Ukupna proizvodnja'] - wind - solar` would fail.
Though if it's important for data post-processing I will set it to None at the returned json.

Sample output is:
```
[{
    'zoneKey': 'HR', 
    'datetime': datetime.datetime(2019, 10, 1, 11, 32, 16, tzinfo=tzutc()), 
    'production': {
        'wind': 12.0, 
        'solar': 33.006206, 
        'unknown': 1205.993794
    },
    'source': 'hops.hr'
}]
```

Reply:
Internally in Electricity Map we do make a distinction: 0 means production reported at 0 (e.g. solar at night), while None means "unknown, not reported, no data available"

Reply:
@jarek :+1: 
Indeed in this case makes sense to return `None` if solar data not available.